### ISSUE TITLE
reverse_dependencies broken?

### ISSUE THREAD

**Issue by andk** [metacpan/metacpan-client#92]

**Comment by andk** [comment]
Oops, sorry, I seem to have clicked on an empty field? Here is another try:

``` text
perl -MMetaCPAN::Client -MData::Dump=dd -e '
dd(MetaCPAN::Client->new()->reverse_dependencies("ElasticSearch"));
'                                                                 
Failed to fetch 'https://fastapi.metacpan.org//reverse_dependencies/dist/ElasticSearch': Not Found at /home/sand/src/perl/repoperls/installed-perls/host/k93msid/v5.27.4/89ad/lib/site_perl/5.27.4/MetaCPAN/Client.pm line 315.
bless({ items => [], total => 0, type => "release" }, "MetaCPAN::Client::ResultSet")
```

**Comment by oalders** [comment]
I think that URL should be `https://fastapi.metacpan.org/v1/reverse_dependencies/dist/Search-Elasticsearch`

**Comment by mickeyn** [comment]
fixed in 2.018000 ... you will have to try it with an existing distribution name though :)
